packages/altspacevr-extras/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/mixed-reality-extension-altspacevr-extras",
-  "version": "0.18.1",
+  "version": "0.18.0-inking.1",
```

Comment:
Please revert these packaging changes before merging to red. We'll release as 0.19 once it's in I expect.

Reply:
Yeah.  This was just from the publish for the GC team to use early in the preview stage.  Thanks for catching this.  I will revert this.  I thought I had done this in a separate branch just for them, but realized that was on the Unity stuff.